ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/priority-fee-input/priority-fee-input.test.js

```diff
@@ -125,6 +125,6 @@ describe('PriorityfeeInput', () => {
     });
     expect(
       screen.queryByText('Priority fee must be greater than 0.'),
-    ).toBeInTheDocument();
+    ).not.toBeInTheDocument();
   });
 });
```

Comment:
Will. be nice to have a test case added for -ve value, since `0` is now a valid value.

Reply:
Fixed here: 62765563566a08d874cf186ef0cd1933b5cad35c